packages/cli/src/lib/mini-oxygen/node.ts

```diff
@@ -118,6 +124,13 @@ export async function startNodeServer({
         body: [
           `View ${options?.appName ?? 'Hydrogen'} app: ${listeningAt}`,
           ...(options?.extraLines ?? []),
+          ...(debug
+            ? [
+                {
+                  warn: `\n\nDebugger listening on ws://localhost:${inspectorPort}`,
```

Comment:
This is important, because the default port that chrome looks for is different than what we default to. I had to manually add `9222` for chrome to see it.

Reply:
In my browser, both 9222 and 9229 are by default added. However, VSCode only uses 9229 by default so I'm going to swap these 2 ports to make 9229 the public one and 9222 the private one for workerd 👍 

<img width="258" alt="image" src="https://github.com/Shopify/hydrogen/assets/1634092/c40fd310-ad3d-4c15-ac50-a8cfef595824">